### ISSUE TITLE
False-negative in `EI_EXPOSE_REP2` if fields are forwarded from one constructor to the other

### ISSUE THREAD

**Issue by Vampire** [spotbugs/spotbugs#3824]
To mitigate `CT_CONSTRUCTOR_THROW`, I changed
```java
public Foo(Collection<String> foos) {
    this.foos = foos;
    this.launcher = LauncherFactory.methodWithThrowsClause();
}
```
to
```java
public Foo(Collection<String> foos) {
    this(foos, LauncherFactory.methodWithThrowsClause());
}

private Foo(Collection<String> foos, Launcher launcher) {
    this.foos = foos;
    this.launcher = launcher;
}
```

In the old version also an instance of `EI_EXPOSE_REP2` was found.
In the new version this is no longer detected while still there.
